operator/pkg/ingress/service.go

```diff
@@ -184,6 +185,12 @@ func (sm *serviceManager) notify(service *slim_corev1.Service) {
 }
 
 func getServiceForIngress(ingress *slim_networkingv1.Ingress) *v1.Service {
+	annotations := make(map[string]string)
+	for annotationKey, annotationValue := range ingress.ObjectMeta.Annotations {
+		if strings.Contains(annotationKey, "service.beta.kubernetes.io") || strings.Contains(annotationPrefix, "service.kubernetes.io") || strings.Contains(annotationPrefix, "cloud.google.com") {
```

Comment:
I can't find annotationPrefix declaration anywhere.

Reply:
I also have few below comments:

- keep the list of allowed annotation prefixes in a slice variable, so that we can add new one easier. This slice can be declared outside of function as well.
- add couple of unit tests for this changes.